### DISCUSSION TITLE
2024-12-10 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#375] [Meetings]
Please review our meeting agenda here:
[UCUM Committee_Meeting Agenda_2024-12-10_FINAL](https://docs.google.com/document/d/1IRbYXS6qEeulmSxdfg4tqPZV1XKJpec7/edit)

[2024-11-04 UCUM Proposals](https://docs.google.com/document/d/1IIB-F0fF08uoESKRM9WGIpUZIrYg3BDaZQFTTx-AfgY/edit?tab=t.0#heading=h.19rlf7f3lcls)

[UCUM Committee_Meeting Summary_2024-08-27_DRAFT.docx](https://docs.google.com/document/d/1_6UPHef0pOUodYu4MPGHo2trGYHBbJwr/edit) 

All active GitHub issues/proposals https://github.com/ucum-org/ucum/issues

@ucum-org/regenstrief
@ucum-org/advisors

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release of LOINC and other events. 

Thank you,

Best Regards,
The Health Data Standards Team
Regenstrief Institute

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release of LOINC and other events. 

Thank you,

Best Regards,
The Health Data Standards Team
Regenstrief Institute
